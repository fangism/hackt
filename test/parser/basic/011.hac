// "011.hac" this should pass, good array declaration
// an example of rejected cast-style arrays is in 010.hac

// eventually type checker will reject this...

int good_array[5];

namespace more_arrays {

int fancy_array[5][3];

int fancier_array[N][0..N][1..M];

}

