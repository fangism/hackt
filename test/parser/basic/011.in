// "011.in" this should pass, good array declaration
// an example of rejected cast-style arrays is in 010.in

// eventually type checker will reject this...

int good_array[5];

namespace more_arrays {

int fancy_array[5][3];

int fancier_array[N][0..N][1..M];

}

