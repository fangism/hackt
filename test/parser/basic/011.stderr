object "N" not found, ERROR!  [12:19]
